lib/src/lifecycle_module.dart

```diff
@@ -297,6 +303,13 @@ abstract class LifecycleModule {
     return completer.future;
   }
 
+  /// Aliased to [unload].
+  ///
+  /// Deprecated: Use the method [unload] instead.
+  @deprecated
+  @override
+  Future<Null> dispose() => unload();
```

Comment:
Discussed in chat. `@protected` doesn't prevent consumers from calling `dispose()` in their subclass of `LifecycleModule`. This is fine.